webcompat/views.py

```diff
@@ -143,6 +144,7 @@ def index():
 
 
 @app.route('/issues')
+@cache_policy(private=True, uri_max_age=86400)
```

Comment:
So far all instances of this decorator use `uri_max_age=86400` -- should that be the default, rather than 3600? Or do you have in mind using the decorator with the default somewhere else?